### DISCUSSION TITLE
请问如何启用mosdns的doh服务？

### DISCUSSION THREAD

**Discussion by Seed680** [kkkgo/PaoPaoDNS#19] [问答和帮助] [answered]
希望mosdns提供doh服务 应该怎么添加配置文件呢 感谢

**Comment by kkkgo** [comment, marked as answer]
参照官方文档即可：https://irine-sistiana.gitbook.io/mosdns-wiki/mosdns-v5/ru-he-pei-zhi-mosdns/fu-wu-qi-cha-jian    
修改/data/mosdns.yaml文件，在末尾添加服务器插件http_server。或者你套一个AdguardHome，点点点
